cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -64,31 +65,31 @@ void MapBuilderBridge::WriteAssets(const string& stem) {
   }
 }
 
-bool MapBuilderBridge::HandleSubmapQuery(
-    cartographer_ros_msgs::SubmapQuery::Request& request,
-    cartographer_ros_msgs::SubmapQuery::Response& response) {
+void MapBuilderBridge::HandleSubmapQuery(
+                                         const std::shared_ptr<::cartographer_ros_msgs::srv::SubmapQuery::Request> request,
+                                         std::shared_ptr<::cartographer_ros_msgs::srv::SubmapQuery::Response> response) {
```

Comment:
Yeah, sorry.  I still haven't configured emacs totally properly.  I'll fix this.